doc/tor.1.txt

```diff
@@ -3430,11 +3430,11 @@ __DataDirectory__**/hashed-fingerprint**::
 
 __DataDirectory__**/approved-routers**::
     Only used by authoritative directory servers. This file lists
-    the status of routers by their identity fingerprint.
-    Each line lists a status and a fingerprint separated by
+    the status of routers by their identity fingerprint or base64-encoded
+    ed25519 public key. Each line lists a status and a fingerprint separated by
```

Comment:
"status and a fingerprint/pubkey"

Reply:
89f4a2500a62dbbbb58205b32e7d32c1b154e37e